tests/OAuth2SponsoredProjectsApiTest.php

```diff
@@ -67,8 +67,8 @@ public function testGelAll()
         );
 
         $content = $response->getContent();
-        $companies = json_decode($content);
-        $this->assertTrue(!is_null($companies));
+        $sponsored_projects = json_decode($content);
```

Comment:
@romanetar we need to update the add and update test to add and check the new attributes

Reply:
done